pipelines/upgrade/08-tests.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- name: run tests
+  hosts:
+    - "{{ forklift_server_name }}"
+  become: yes
+  vars_files:
+    - ../vars/upgrade_base.yml
+    - ../vars/repos_staging.yml
+  vars:
+    bats_output_dir: "/root/bats_results_upgrade1"
```

Comment:
What about `bats_results_proxy_n-1_upgrade` ? Or some such naming to make it easier when looking through the data to remind ourselves what each means.